sysl2/sysl/seqs/visitor.go

```diff
@@ -210,7 +238,22 @@ func (v *SequenceDiagramVisitor) Visit(e Element) error {
 
 	switch t := e.(type) {
 	case *EndpointCollectionElement:
-		return v.visitEndpointCollection(t)
+		v.visitEndpointCollection(t)
+		for bbKey, bbVal := range t.blackboxes {
+			if bbVal.valueType >= BbApplication && bbVal.visitCount == 0 {
+				var scope string
+				switch bbVal.valueType {
+				case BbApplication:
+					scope = fmt.Sprintf("' %s :: '", v.currentApp)
+				case BbEndpointCollection:
+					scope = fmt.Sprintf("' %s :: %s'", v.currentApp, t.title)
+				default:
+					scope = ""
+				}
+				fmt.Fprintf(v.stdout, fmt.Sprintf("blackbox '%s' not hit in app %s\n", v.currentApp, t.title))
```

Comment:
I still don't understand the need to output to stdout. If it's just for testing logs, this isn't the same output as the log anyway. You should really be [hooking logrus](https://github.com/sirupsen/logrus#testing) to observe its behaviour. Unfortunately, this relies on passing a Logger instance around rather than relying on the standard logger as we do. This might be a good opportunity to change this.

Reply:
Sure will have a look.

Reply:
Taken care in the latest commit.